src/acceptanceTest/java/uk/gov/hmcts/futurehearings/hmi/acceptance/common/security/OAuthTokenGenerator.java

```diff
@@ -1,46 +1,49 @@
 package uk.gov.hmcts.futurehearings.hmi.acceptance.common.security;
 
-import static io.restassured.RestAssured.expect;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
 import io.restassured.http.ContentType;
 import io.restassured.response.Response;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.http.HttpStatus;
 
+import static io.restassured.RestAssured.expect;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 @Slf4j
 public class OAuthTokenGenerator {
```

Comment:
class can be declared as final class as this class is having only one single private constructor

Reply:
Out of scope